include/vcpkg/build.h

```diff
@@ -220,9 +220,19 @@ namespace vcpkg
 
     StringLiteral to_string_locale_invariant(const BuildResult build_result);
     LocalizedString to_string(const BuildResult build_result);
-    LocalizedString create_user_troubleshooting_message(const InstallPlanAction& action,
-                                                        const VcpkgPaths& paths,
-                                                        Optional<Path>&& issue_body = nullopt);
+    LocalizedString create_user_troubleshooting_message(const InstallPlanAction& action, const VcpkgPaths& paths);
+    inline void print_user_troubleshooting_message(const InstallPlanAction& action,
+                                                   const VcpkgPaths& paths,
+                                                   Optional<Path>&& issue_body)
+    {
+        msg::println_error(create_user_troubleshooting_message(action, paths));
+        if (issue_body)
+        {
+            // Intentionally not using `println_warning` here to avoid `warning: ` prefix
```

Comment:
```suggestion
```

I think it's clear without the comment: it's printing a troubleshooting message, not a warning. Could go either way though.